plugins/ohifv3/extensions/monai-label/src/components/SettingsTable.tsx

```diff
@@ -39,16 +50,28 @@ export default class SettingsTable extends Component {
     const url = evt.target.value;
     this.setState({ url: url });
     CookieUtils.setCookie('MONAILABEL_SERVER_URL', url);
+    console.log('Settings onBlurSeverURL', url);
+  };
+
+  onConnect = () => {
+    const url = document.getElementById('monailabelServerURL').value;
```

Comment:
you could use Refs

this.inputRef = React.createRef()

assign on input 

then const url = this.inputRef.current.value;